src/AutoRest.CSharp/AutoRest.CSharp.csproj

```diff
@@ -18,7 +18,8 @@
 
 
   <ItemGroup>
-    <PackageReference Include="Azure.Core" Version="1.25.0" />
+    <PackageReference Include="Azure.Core" Version="1.26.0" />
```

Comment:
In order for us to test this version bump we need to update [here](https://github.com/Azure/autorest.csharp/blob/feature/v3/src/AutoRest.CSharp/Common/AutoRest/Plugins/CSharpProj.cs#L36) and regen the entire library by running `./eng/Generate.ps1` from the root of the repo.